yoctoproject/vscode-bitbake
# Feat: Add reference provider

Our existing `onDefinition` handler is doing well in most cases but doesn't seem to be able to handle single instances alone from my current understanding and experiments. Adding the `referenceProvider` with proper implementation can allow the `Go to definition` to work as expected on single instances.

Check the internal tracking ticket for more details on how I reached the conclusion: 13914

How to verify:
Create your variable like this `MYVAR = '123'` and make sure it is unique in all files. Then, trigger `Go to definition` on it. It should briefly highlight the variable and should not show anything like `No reference found on 'MYVAR'`
